components/nofrendo/nes/nes.h

```diff
@@ -45,7 +45,7 @@
 #ifdef PAL
 #define  NES_REFRESH_RATE     50
 #else /* !PAL */
-#define  NES_REFRESH_RATE     60
+#define  NES_REFRESH_RATE     40
```

Comment:
No, a NTSC NES runs at 60FPS. Not sure why you did this, but please keep the defines that documents basic facts in place.

Reply:
I think he had watchdog problems, just like me. But this is fixed by changing the compilation to PAL.